kedro-datasets/tests/ibis/test_table_dataset.py

```diff
@@ -112,10 +100,10 @@ def test_exists(self, table_dataset, dummy_table):
         table_dataset.save(dummy_table)
         assert table_dataset.exists()
 
-    @pytest.mark.parametrize("load_args", [{"filename": True}], indirect=True)
-    def test_load_extra_params(self, table_dataset_from_csv, load_args):
-        """Test overriding the default load arguments."""
-        assert "filename" in table_dataset_from_csv.load()
+    def test_load_extra_params(self):
+        """Test overriding the load arguments."""
+        table_dataset = TableDataset(table_name="test", load_args={"database": "test"})
+        assert "database" in table_dataset._load_args
```

Comment:
```suggestion
    @pytest.mark.parametrize("load_args", [{"filename": True}], indirect=True)
    def test_load_extra_params(self, table_dataset, load_args):
        """Test overriding the default load arguments."""
        assert table_dataset._load_args["database"] == "test"
```

or even

```suggestion
    @pytest.mark.parametrize("load_args", [{"filename": True}], indirect=True)
    def test_load_extra_params(self, table_dataset, load_args):
        """Test overriding the default load arguments."""
        for key, value in load_args.items()
            assert table_dataset._load_args[key] == value
```

if you want to be extra consistent with the other dataset tests